releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,15 @@
 ## Microsoft.Extensions.Configuration.AzureAppConfiguration
 ### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration)
 
+### 3.0.0 - February 19, 2020
+* Added the following method to allow users to override `ConfigurationClientOptions`. These can be used to update the default retry options or configure a proxy for connecting to Azure App Configuration.
```

Comment:
The usage of the word 'or' makes it seem like those are the only two possible use cases. We could change to something like "This enables customization on the underlying App Configuration client including modifying retry options and configuring proxy settings."

Reply:
In addition to what Jimmy said, you can add a link to the [issue](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/106) now.